git_repo/services/ext/gitlab.py

```diff
@@ -16,8 +16,8 @@ class GitlabService(RepositoryService):
     fqdn = 'gitlab.com'
 
     def __init__(self, *args, **kwarg):
-        self.gl = gitlab.Gitlab(self.url_ro)
-        super(GitlabService, self).__init__(*args, **kwarg)
+        super().__init__(*args, **kwarg)
+        self.gl = gitlab.Gitlab(self.url_ro, ssl_verify=not self.insecure)
```

Comment:
is any of the underlying library warn about this?
and maybe have you got an opinion about yelling about insecure connections?